### PR TITLE
Add support to pipSize

### PR DESCRIPTION
**Pull request by qingweibinary**
Sorry for the messy commit history

### CODE REVIEW

File: src/config/updateAxis.js [hunk 1]
```diff
@@ -0,0 +1 @@
+export default (axis, options) => axis.update(options);
```
Comment (borisyankov) [inline src/config/updateAxis.js]: Isn't this an overkill? :)
updateAxis(axis, options)
vs
axis.update(options)
:)

Comment (qingweibinary) [inline src/config/updateAxis.js]: Yes I did it for consistency, will make it inline

**Comment by qingweibinary** [comment]
I'll fix the test, a minute

**Comment by qingweibinary** [comment]
Seems it's not build fail, why coverall fail? because of percentage dropped?

**Pull request merged**